torchgeo/datasets/nlcd.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""National Agriculture Imagery Program (NAIP) dataset."""
+
+from typing import Any, Dict, Optional
+
+import matplotlib.pyplot as plt
+
+from .geo import RasterDataset
+
+
+class NLCD2016TreeCanopy(RasterDataset):
+    """National Land Cover Database 2016 (NLCD2016) - Tree Canopy dataset.
+
+    The `National Land Cover Database <https://www.mrlc.gov/>`_ provides 30m tree
```

Comment:
I would link to the tree canopy page, not the NLCD page

Reply:
```suggestion
    The `Multi-Resolution Land Characteristics (MRLC) Consortium <https://www.mrlc.gov/>`_ provides 30m tree
```

Reply:
Here's the specific page for reference https://www.mrlc.gov/data/nlcd-2016-usfs-tree-canopy-cover-conus